docs/standard/linq/comparison-xpath-linq-xml.md

```diff
@@ -0,0 +1,107 @@
+---
```

Comment:
I didn't expect to see new articles on this PR @jameshkramer. By design?

Reply:
That article is added in br26. I think we should merge br26 before br28.

Reply:
Sounds good.